app/notifiers/publishing_api_notifier.rb

```diff
@@ -11,12 +11,8 @@ def initialize(tag)
     @tag = tag
   end
 
-  def send_single_tag_to_publishing_api
-    if tag.draft?
-      publishing_api.put_draft_content_item(presenter.base_path, presenter.render_for_publishing_api)
-    else
-      publishing_api.put_content_item(presenter.base_path, presenter.render_for_publishing_api)
-    end
+  def write_content
```

Comment:
I don't think that adds anything here - it's called in the pattern:
PublishingAPINotifier.new(tag).write content, or as part of PublishingAPINotifier.send_to_publishing_api(tag), where it's obviosu that it's doing other stuff too like dependent tags and root topics/pages.